docs/source/release_notes/release-notes.rst

```diff
@@ -46,6 +46,32 @@ develop
          - KVS migration CLI will now clear the checkpoint tables that are required while the migration is in progress but not after the migration is complete.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1927>`__)
 
+    *    - |new|
+         - Added support for live-reloading sweep configs.
+
+           There's a new config block, the `atlas-runtime`, which is live-reloaded. Currently, it accepts the config options:
+
+             - `enableSweep`
+             - `getSweepPauseMillis`
+             - `getSweepReadLimit`
+             - `getSweepCandidateBatchHint`
+             - `getSweepDeleteBatchHint`
+
+           And the deprecated config options:
+
+             - `getSweepBatchSize`
+             - `getSweepCellBatchSize`
+
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1976>`__)
```

Comment:
I think for the above it might be better to point users to the existing sweep config docs, saying that the options/way you specify it is essentially the same.